src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -81,11 +81,10 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit settings: Settings, id: String): Try[Unit] = {
     log.info(s"[$id] Finalizing deposit")
-    implicit val bagStoreBase = BagStoreBase(settings.bagStoreBaseDir, settings.bagStoreBaseUrl)
+    implicit val bagStoreSettings = settings.bagStoreSettings
```

Comment:
You're right; I left this `Option` in in my refactoring (https://github.com/vesaakerman/easy-sword2/pull/4).